plugin/trino-delta-lake/src/test/resources/deltalake/clone_merge/clone_merge_cloned/README.md

```diff
@@ -0,0 +1,14 @@
+Data generated using OSS Delta Lake 3.3.0 and Trino 469:
+
+```sql
+-- Spark>
+CREATE TABLE clone_merge_cloned SHALLOW CLONE clone_merge_source;
+
+-- Trino>
+UPDATE clone_merge_cloned set v = 'update1' where id in (2,4);
```

Comment:
Can you move this operation to test itself and add MERGE operation as name sugest?

Reply:
I am not sure understand correctly, do you mean replacing the update statement to merge statement?

Reply:
First move `UPDATE` to testcase itself, it's done using Trino so I suppose it should be possible. In this way test will test `UPDATE` and `SELECT`, also it will be easier to debug it if needed. 
Later you could add also `MERGE` statement to test, or create additional test based on the same resources with `MERGE` query.